lib/web_ui/lib/src/engine/compositor/fonts.dart

```diff
@@ -5,7 +5,7 @@
 part of engine;
 
 const String _robotoUrl =
```

Comment:
Does this cause any issue if you're on http? (I wouldn't think so...)

Reply:
The other fonts load in HTTPS so I presume not. I am currently building this to double check.

Reply:
This worked on HTTP and HTTPS (see https://testchillers-dot-flutter-dashboard.appspot.com/#/ for a demo of just the Skia site - There are some issues from the icons and I think its due to me doing a manual deploy)